src/components/ui/drop-files/__mocks__/createLocalFile.ts

```diff
@@ -0,0 +1,8 @@
+export async function createLocalFile(fileSrc: string): Promise<File> {
+  const response = await fetch(fileSrc);
+  const blob = await response.blob();
+  const splitted = fileSrc.split('/');
+  const fileName = splitted.pop() || 'local-image';
+
+  return new File([blob], fileName, { type: blob.type });
+}
```

Comment:
Fiquei na dúvida o que seria o `local-image`. Outro ponto iria sugerir a seguinte melhoria:

```suggestion
import { parse } from 'node:path';

export async function createLocalFile(src: string): Promise<File> {
  const blob = await fetch(src).then(response => response.blob());    
  const filename = parse(src).base() || 'local-image';

  return new File([blob], filename, { type: blob.type });
}
```

Reply:
Só um nome genérico para o nome do arquivo Rafa, essa função é só para uso no storybook, por isso ela está na pasta de mocks